frontends/mit-open/src/common/utils.ts

```diff
@@ -0,0 +1,13 @@
+const getSearchParamMap = (urlParams: URLSearchParams) => {
```

Comment:
I think it would be worth adding a few simple tests for this.

Reply:
added some tests

Reply:
tests added